doc/Working_with_Iris_Cubes.ipynb

```diff
@@ -0,0 +1,383 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%%opts BoxWhisker [xrotation=25 bgcolor='w']\n",
+    "hv.Layout([cube.to.box(d, mdims=[]) for d in ['time', 'realization']])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This also gives us access to other statistical plot types, e.g. with the ``seaborn`` library installed we also have access to the ``Distribution`` Element which visualizes the data as a kernel density estimate. In this way we can visualize how the distribution of surface temperature values varies over time and the model realizations. We do this by ignoring the 'latitude' and 'longitude' in the list of mdims, generating a lower dimensional view into the data and laying out 'time' and 'realization' in a GridSpace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%opts GridSpace [shared_xaxis=True] \n",
+    "%opts Distribution [bgcolor='w' show_grid=False xticks=[220, 300]]\n",
+    "try:\n",
+    "    import seaborn\n",
+    "    grid = cube.to.distribution(mdims=['realization', 'time']).grid()\n",
+    "except:\n",
+    "    grid = None\n",
+    "grid"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reducing the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So far all the examples we have seen have displayed all the data in some way or another. Another way to explore a dataset is to explicitly reduce the dimensionality or select subregions of a dataset. There are two main ways to do this, either we explicitly select a subset of the data or we collapse a dimension using an aggregation function, e.g. by computing a mean along a particular dimension."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Selecting slices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the ``select`` method we can easily select ranges of coordinates in the Cube. The select method does not however know that latitude and longitude are cyclic, so instead we can select regions at both ends of the prime meridian ($0^\\circ$ longitude) and overlay them. In this way we can stitch together multiple cubes or simply view a subregion specific subregions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "northern = cube.select(latitude=(25, 75))\n",
+    "(northern.select(longitude=(260, 305)).to(gv.Image, geo_dims) *\n",
```

Comment:
The idea was to leave a gap to demonstrate you can overlay subregions of interest even if they are not contiguous.